src/components/node.js

```diff
@@ -43,6 +43,7 @@ export type INode = {
 type INodeProps = {
   data: INode,
   id: string,
+  disableGraphKeyHandlers?: boolean,
```

Comment:
Is this controlling just the shift + background combo? Could we perhaps then use a more distinct name? Or should we maybe consider nesting these interaction-setting flags under some kind of Object-type prop? 
cc @PankajTrifacta

Reply:
@hodos I actually just switched this from a separate prop to use this one, since it already existed. It's also used for enabling/disabling interactions like copy/paste, delete key events and I just repurposed it to also be used for the shift key combinations. I can make this an object though, if we think that would be better!

Reply:
ahh kk, so it covers a bunch of interactions at once? yah, I'm not sure, if we want to separate these under several keys in an object-type prop, but I guess it's fine for now ?

Reply:
Yeah if we do decide we want to make it an object, it should be fairly easy to replace since aside from all the updates I'm making, it's only used in the `componentDidMount/unmount` methods